documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1741,6 +1741,39 @@ endif::product[]
 
 |===
 
+[id="oracle-xml-types"]
+=== XML type
+ifdef::community[]
+[NOTE]
+====
+Support for `XMLTYPE` is currently in incubating state, that is, the exact semantics, configuration options and so forth might change in future revisions, based on feedback we receive.
+Please let us know if you encounter any problems while using these data types.
+====
+endif::community[]
+ifdef::product[]
+[IMPORTANT]
+====
+Use of the `XMLTYpE` with the {prodname} Oracle connector is a Technology Preview feature only.
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
+====
+endif::product[]
+The following table describes how the connector maps XMLTYPE data types.
+
+.Mappings for Oracle XMLTYPE data types
+[cols="20%a,15%a,55%a",options="header"]
+|===
+|Oracle Data Type
```

Comment:
```suggestion
|Oracle data type
```
We use sentence-style capitalization in table headings downstream. I know that this isn't consistently applied upstream, so we can either let it go here for now, or make the change and give me the task to do a thorough scrubbing of the doc set to apply the formatting.